Resources/Prototypes/_NF/Entities/Mobs/NPCs/emotionalsupportanimals.yml

```diff
@@ -0,0 +1,116 @@
+- type: entity
+  id: BaseEmotionalGhost
+  abstract: true
+  components:
+  - type: GhostTakeoverAvailable
+  - type: GhostRole
+    name: ghost-role-information-emotional-support-name
+    description: ghost-role-information-emotional-support-description
+    rules: ghost-role-information-emotional-support-rules
+    makeSentient: true
+    allowSpeech: true
+    allowMovement: true
+    prototype: EmotionalSupportPet
```

Comment:
No need for this ``prototype: EmotionalSupportPet`` its used in frontier for whitelist